src/utils/navigation.js

```diff
@@ -63,7 +63,6 @@ export const findPage = (url) => {
     if (host === githubHost) {
       const meta = pathToMeta(pathname)
       if (
-          meta.org === page.org &&
```

Comment:
hmm, so the org is allowed to be different? You sure its the right fix? Maybe meta.org or page.org parsing is wrong?